### PR TITLE
Add Brent Baude's Mac Update post

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add the post from Brent about Podman on Macs (and Windows) status.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan @mheon @lsm5 @pthomas PTAL

**Comment by mheon** [comment]
LGTM

**Review by lsm5 — APPROVED** [review]
LGTM.

**Comment by lsm5** [comment]
err, before we merge, i think we should also include our matrix / discord channels. They may be bridged, but people have their preferences. Not a blocker though.

**Comment by lsm5** [comment]
See: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w#-in-the-video-33 for discord and matrix bridge info.

File: _posts/2021-09-06-podman-on-macs.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### User Experience on Windows 
+We currently support the Windows platform with a remote client that can be downloaded from our [GitHub releases page](https://github.com/containers/podman/releases).  That remote client requires a Linux server with Podman and its service running.  We also have user reports that running Podman in WSL is quite tenable.  Consider the WSL option if you do not have available Linux servers with Podman installed.
+
+We intend to develop a desktop for the Mac and Windows experience for Podman.  Early design work is under consideration.  No timeline has been identified yet.
+
+### Questions?
+Remember, our development team can be found on [libera IRC](https://libera.chat/) in the #podman channel. You can also get in touch with us via our [project page](https://github.com/containers/podman) by opening issues, PR’s and discussions. We love to hear from people!
```
Comment (lsm5) [inline _posts/2021-09-06-podman-on-macs.md]: Remember, our development team can be found in our [Matrix room](https://matrix.to/#/#podman:matrix.org) which has been bridged to the #podman channel on [libera IRC](https://libera.chat/) as well as our [Discord server](https://discord.gg/x5GzFF6QH4). You can also get in touch with us via our [project page](https://github.com/containers/podman) by opening issues, PR’s and discussions. We love to hear from people!

Comment (lsm5) [reply _posts/2021-09-06-podman-on-macs.md]: I've set the discord link to never expire

Comment (TomSweeneyRedHat) [reply _posts/2021-09-06-podman-on-macs.md]: Thanks, just updated

**Comment by TomSweeneyRedHat** [comment]
Pushing this through.

**Pull request merged**